builtin/credential/aws/path_config_client.go

```diff
@@ -77,15 +82,29 @@ func (b *backend) pathConfigClient() *framework.Path {
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.CreateOperation: &framework.PathOperation{
 				Callback: b.pathConfigClientCreateUpdate,
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationVerb:   "configure",
+					OperationSuffix: "client",
+				},
 			},
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.pathConfigClientCreateUpdate,
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationVerb:   "configure",
+					OperationSuffix: "client",
+				},
 			},
 			logical.DeleteOperation: &framework.PathOperation{
 				Callback: b.pathConfigClientDelete,
+				DisplayAttrs: &framework.DisplayAttributes{
+					OperationSuffix: "client-configuration",
```

Comment:
It feels a bit inconsistent that the suffix and verb on the last two differ from the first two even though they are all against the `config/client` endpoint.

Reply:
Similar to https://github.com/hashicorp/vault/pull/19540#discussion_r1156498441, the team consensus seems to be to go with "configure" for the update/create commands to make it easier to read in code & more aligned with the documentation nomenclature.